app/src/main/java/au/com/bluedot/minimalintegration/AppGeoTriggerReceiver.java

```diff
@@ -0,0 +1,45 @@
+package au.com.bluedot.minimalintegration;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.Looper;
+import android.widget.Toast;
+import au.com.bluedot.point.net.engine.GeoTriggeringEventReceiver;
+import au.com.bluedot.point.net.engine.ZoneEntryEvent;
+import au.com.bluedot.point.net.engine.ZoneExitEvent;
+import au.com.bluedot.point.net.engine.ZoneInfo;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+
+public class AppGeoTriggerReceiver extends GeoTriggeringEventReceiver {
+    private final Handler handler = new Handler(Looper.getMainLooper());
```

Comment:
Shouldn't be any need for this, broadcasts are always received on the main thread.